js/main.js

```diff
@@ -62,7 +66,8 @@ document.querySelector('#book-form').addEventListener('submit', (e) => {
     alert('Title and Author fields must be filled out');
```

Comment:
- [ ] One of Javascript best-practice state that `Do not use window.alert() or window.confirm(). It’s ugly, impossible to style, stops code execution, and displays differently on different browsers. Use custom modal instead.` [reference](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). Kindly let's go ahead and remove this alert message or display this validation in an HTML tag next to the form that raised a validation error.